requirements.txt

```diff
@@ -73,3 +73,4 @@ django-test-migrations==1.1.0
 djangosaml2==1.3.3
 drf-spectacular==0.19.0
 django-ratelimit==3.0.1
+XlsxWriter==3.0.1
```

Comment:
We have already have a dependency on module `openpyxl`.

Reply:
Yes, we should have only one library. I will have a look which one to keep.

Reply:
The Excel export is now using openpyxl and I removed XlsxWriter from the requirements.